simulator/task_manager.go

```diff
@@ -43,29 +46,96 @@ func (m *TaskManager) init(r *Registry) {
 			m.Description = esx.Description
 		}
 	}
+
+	if m.MaxCollector == 0 {
+		m.MaxCollector = 1000
```

Comment:
We should probably make this a well-documented constant.

Reply:
Added a `var maxCollectors`. Looks like VC's default is 32 and can be changed w/ OptionManager, but don't have a use-case to support that yet.

Reply:
Should it be a constant that we can adjust as needed? Is there any reason it _should_ be a variable?

Reply:
If we end up with a use-case to support changing the max via OptionManager, then we'd need a variable. But I just clumped it into the existing set of non-exported `var (...)`, do we gain anything by making it a constant?